SignalServiceKit/src/Storage/TSStorageManager.m

```diff
@@ -22,8 +23,41 @@
 NSString *const TSStorageManagerExceptionName_CouldNotCreateDatabaseDirectory
     = @"TSStorageManagerExceptionName_CouldNotCreateDatabaseDirectory";
 
-#pragma mark -
+void runSyncRegistrationsForPrimaryStorage(OWSStorage *storage)
```

Comment:
nit: the term `primary` feels kind of oblique given there is no longer "secondary" storage.

Reply:
Removed.